pkg/apis/tenancy/v1alpha1/types.go

```diff
@@ -74,18 +74,36 @@ type ClusterWorkspaceSpec struct {
 
 	// type defines properties of the workspace both on creation (e.g. initial
 	// resources and initially installed APIs) and during runtime (e.g. permissions).
+	// If no type is provided, the default type for the workspace in which this workspace
+	// is nesting will be used.
 	//
-	// The type is a reference to a ClusterWorkspaceType in the same workspace
-	// with the same name, but lower-cased. The ClusterWorkspaceType existence is
-	// validated at admission during creation, with the exception of the
-	// "Universal" type whose existence is not required but respected if it exists.
-	// The type is immutable after creation. The use of a type is gated via
+	// The type is a reference to a ClusterWorkspaceType in the listed workspace, but
+	// lower-cased. The ClusterWorkspaceType existence is validated at admission during
+	// creation. The type is immutable after creation. The use of a type is gated via
 	// the RBAC clusterworkspacetypes/use resource permission.
 	//
 	// +optional
-	// +kubebuilder:default:="Universal"
+	Type ClusterWorkspaceTypeReference `json:"type,omitempty"`
+}
+
+// ClusterWorkspaceTypeReference is a globally unique, fully qualified reference to a
+// cluster workspace type.
+type ClusterWorkspaceTypeReference struct {
+	// name is the name of the ClusterWorkspaceType
+	//
+	// +required
 	// +kubebuilder:validation:Pattern=`^[A-Z][a-zA-Z0-9]+$`
-	Type string `json:"type,omitempty"`
+	Name string `json:"name,omitempty"`
+
+	// path is an absolute reference to the workspace that owns this type, e.g. root:org:ws.
+	//
+	// +required
+	// +kubebuilder:validation:Pattern:="^root(:[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$"
+	Path string `json:"path,omitempty"`
+}
+
+func (r ClusterWorkspaceTypeReference) String() string {
+	return r.Path + "|" + r.Name
```

Comment:
do we leak this notation to users anywhere else? Alternatively, "%q in workspace %q". Would use the later.